calnex/cert/cert_test.go

```diff
@@ -0,0 +1,204 @@
+/*
+Copyright (c) Facebook, Inc. and its affiliates.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cert_test
```

Comment:
Change package name to `cert` and then errors don't need to be exposed.

Reply:
I think it's cleaner to test the surface of a package where possible rather than reaching into the implementation. In this instance either works, but given that exporting the errors makes some sense there's nothing to be gained by having the tests in the same package.

Reply:
I'd argue it's common to have them separate too, but I'm a sucker for consistency and if everything else is going to be in-package so should this; I'll change it.